config/database.yml

```diff
@@ -81,5 +84,5 @@ test:
 production:
   <<: *default
   database: WarAndPiece_production
-  username: WarAndPiece
-  password: <%= ENV['WARANDPIECE_DATABASE_PASSWORD'] %>
+  # username: WarAndPiece
+  # password: <%= ENV['WARANDPIECE_DATABASE_PASSWORD'] %>
```

Comment:
Don't worry about committing any changes to this file. The file should work as it is once you run this line in vagrant: 

```
sudo -u postgres createuser --createdb vagrant
```

Reply:
Thanks! I reverted the changes, and ran the sudo -u  command. However, I am now getting an error about "insufficient privilege" when running `bundle exec rspec`. I wonder if the issue may be only on my local environment or for everyone. I'm not sure whether this concerns only this Pull Request or if I should reach out to teammates on Slack, or just wait for our next call?